docs/core/store.md

```diff
@@ -240,17 +240,15 @@ The SDK is in the process of transitioning to use the types listed here as the d
 
 An interface providing only the basic CRUD functionality (`Get`, `Set`, `Has`, and `Delete` methods), without iteration or caching. This is used to partially expose components of a larger store, such as a `flat.Store`.
 
-### Flat Store
+### Root Store
 
-`flat.Store` is the new default persistent store, which internally decouples the concerns of state storage and commitment scheme. Values are stored directly in the backing key-value database (the "storage" bucket), while the value's hash is mapped in a separate store which is able to generate a cryptographic commitment (the "state commitment" bucket, implmented with `smt.Store`).
+`RootStore` is the new interface for the main client store, replacing the function of `MultiStore`. It internally decouples the concerns of state storage and state commitment: values are stored and read directly from the backing key-value database, but are also mapped in a logically separate *state-commitment* store which generates cryptographic proofs.
```

Comment:
1. Let's add a link to ADR-40 here
2. Since we are using multiple SMT instances , maybe we should go back and use _multistore_ name rather than _root store_.